types/src/chainspec/highway_config.rs

```diff
@@ -13,6 +13,17 @@ use crate::{
     TimeDiff,
 };
 
+/// Configuration values relevant to the `PerformanceMeter`, a component of Highway deciding when
+/// to change the round exponent.
+#[derive(Copy, Clone, PartialEq, Eq, Serialize, Deserialize, Debug, Default)]
+#[cfg_attr(feature = "datasize", derive(DataSize))]
+// Disallow unknown fields to ensure config files and command-line overrides contain valid keys.
+#[serde(deny_unknown_fields)]
+pub struct PerformanceMeterConfig {
+    /// The number of recent blocks to take into account when measuring performance.
+    pub blocks_to_consider: u64,
+}
+
```

Comment:
Old thing, don't reintroduce.

Reply:
Removed in 080a969198464c157308f2b8dffd72450c0aece4